_posts/2024-06-21-this-week-in-rails.markdown

```diff
@@ -45,8 +45,8 @@ Minitest supports passing the failure message as a callable, which allow us to d
 Some of these failure messages can be a bit costly to generate, particularly when inspecting very large objects or when accessing the AST of procs.
```

Comment:
Kept "Minitest" same as before:
- Its start of a sentence + refers to it as a library